benchmarks/testminer.py

```diff
@@ -112,10 +112,39 @@ def __init__(self, base_url, username, password):
         self.build_number = int(spl.path.split("/")[4])
         self.username = username # API username
         self.password = password # API token
-        self.jenkins = jenkins.Jenkins(self.url, username=self.username, password=self.password)
-        self.jenkins_job = self.jenkins[self.job_name]
-        self.jenkins_build = self.jenkins_job.get_build(self.build_number)
-        self.jenkins_artifacts = self.jenkins_build.get_artifact_dict()
+
+        self.__jenkins__ = None
+        self.__jenkins_job__ = None
+        self.__jenkins_build__ = None
+        self.__jenkins_artifacts__ = None
+
+    @property
+    def jenkins(self):
+        if self.__jenkins__ is None:
+            self.__jenkins__ = jenkins.Jenkins(
+                self.url,
+                username=self.username,
+                password=self.password
+            )
+        return self.__jenkins__
+
+    @property
+    def jenkins_job(self):
+        if self.__jenkins_job__ is None:
+            self.__jenkins_job__ = self.jenkins[self.job_name]
```

Comment:
For some reason this fails on jenkinsapi 0.2.32 (at least it fails for me). I didn't debug further. The solution was to downgrade to 0.2.30